cli/hooks.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/python
 from __future__ import print_function
-from cli.utils import Utils
 import os
 import contextlib
 import sys
+from sets import Set
```

Comment:
I don not see the usage of set in this file . Do we need to add the import "sets"